src/test/regress/bin/normalized_tests.lst

```diff
@@ -1,40 +1,42 @@
 # List of tests whose output we want to normalize, one per line
-multi_alter_table_add_constraints
-multi_alter_table_statements
-foreign_key_to_reference_table
+custom_aggregate_support
 failure_copy_on_hash
-failure_savepoints
-foreign_key_restriction_enforcement
 failure_real_time_select
+failure_savepoints
 failure_vacuum
+foreign_key_restriction_enforcement
+foreign_key_to_reference_table
+intermediate_results
 isolation_citus_dist_activity
+multi_alter_table_add_constraints
+multi_alter_table_statements
+multi_copy
+multi_create_table_constraints
+multi_explain
+multi_foreign_key
+multi_generate_ddl_commands
+multi_having_pushdown
 multi_insert_select
 multi_insert_select_conflict
-multi_multiuser
-multi_name_lengths
-multi_partition_pruning
-multi_subtransactions
-multi_modifying_xacts
-multi_insert_select
-sql_procedure
-multi_reference_table
-multi_create_table_constraints
-
-# the following tests' output are
```

Comment:
These were splitted in a way that the following were only added here because of EXPLAIN output differences of the executor type. (`Adaptive` vs `Real-time`  or `Adaptive` vs 'Router'). It was done because it'd simplify things when we remove the support for real-time/router queries. 

Is there a reason you changed the order?

Reply:
Ping on this as well.

Reply:
There were duplicates & adding to a list is easier when it's alphabetical. I agree with @pykello that we should remove `normalized_tests.lst` & have a branch that does that, but it needs to figure out how to exclude vanillatest from normalization